lib/tool_belt/template.rb

```diff
@@ -3,12 +3,10 @@
 
 module ToolBelt
   module Template
-    def self.render(template, vars)
-      ERB.new(template, trim_mode: '-').result(OpenStruct.new(vars).instance_eval { binding })
-    end
-
     def self.render_file(filename, context)
-      self.render(File.read(filename), context)
+      erb = ERB.new(File.read(filename), trim_mode: '-')
+      erb.filename = File.expand_path(filename) if filename
```

Comment:
When can `filename` be false-y? It managed to read the file in the line before, so it's pretty sure set?

Reply:
That's a leftover from when I added a `filename` parameter to `render` before. Then I realized this method was the only place that used it so I dropped the `render` method instead. Will drop.

Reply:
:+1: